zammad/templates/statefulset.yaml

```diff
@@ -1,482 +0,0 @@
-apiVersion: apps/v1
```

Comment:
As mentioned above this might still be needed if "attachments to storage option" is used.

Reply:
This should be solved by requiring an external claim.